src/ssl.c

```diff
@@ -55284,6 +55327,85 @@ int wolfSSL_CTX_set_ecdh_auto(WOLFSSL_CTX* ctx, int onoff)
     (void)onoff;
     return WOLFSSL_SUCCESS;
 }
+
+/**
+ * set security level(wolfSSL doesn't suppor security level)
+ * @param ctx  a pointer to WOLFSSL_EVP_PKEY_CTX structure
+ * @param level security level
+ */
+void wolfSSL_CTX_set_security_level(WOLFSSL_CTX* ctx, int level)
+{
+    WOLFSSL_ENTER("wolfSSL_CTX_set_security_level");
+    (void)ctx;
+    (void)level;
+}
+/**
+ * get security level(wolfSSL doesn't suppor security level)
+ * @param ctx  a pointer to WOLFSSL_EVP_PKEY_CTX structure
+ * @return always 0(level 0)
+ */
+int wolfSSL_CTX_get_security_level(const WOLFSSL_CTX* ctx)
+{
+    WOLFSSL_ENTER("wolfSSL_CTX_get_security_level");
+    (void)ctx;
+    return 0;
+}
+
+#ifndef NO_WOLFSSL_STUB
+
+/**
+ * validate the algorithm parameters of the key-pair
+ * @param ctx  a pointer to WOLFSSL_EVP_PKEY_CTX structure
+ * @return WOLFSSL_FAILURE for now
+ */
+int wolfSSL_EVP_PKEY_param_check(WOLFSSL_EVP_PKEY_CTX* ctx)
+{
+    WOLFSSL_STUB("wolfSSL_EVP_PKEY_param_check");
+    (void)ctx;
+    return WOLFSSL_FAILURE;
+}
+
+/**
+ * get call back function for psk session use
```

Comment:
Should this comment be "Set callback" instead of "get"?  Looks like a setter function.

Reply:
Yes. That is true. Thanks